rules/aip0121/resource_must_support_get.go

```diff
@@ -26,11 +26,10 @@ import (
 var resourceMustSupportGet = &lint.ServiceRule{
 	Name: lint.NewRuleName(121, "resource-must-support-get"),
 	LintService: func(s *desc.ServiceDescriptor) []lint.Problem {
-		problems := []lint.Problem{}
-		// Add the empty string to avoid adding multiple nil checks.
-		// Just say it's valid instead.
-		resourcesWithGet := stringset.New("")
```

Comment:
FYI @toumorokoshi we don't need to init with empty string, we can init as `nil` and the `stringset.Set` methods will handle when the receiver is `nil`/empty https://go.dev/play/p/2BXM0MTu1i7